src/order/ideal.lean

```diff
@@ -219,20 +239,22 @@ I.mem_of_le (sup_le hx hy) h_mem
 
 end semilattice_sup
 
-section semilattice_sup_bot
-variables [semilattice_sup_bot P] (I J K : ideal P)
+section semilattice_sup_ideal_inter_nonempty
 
-/-- The intersection of two ideals is an ideal, when `P` has joins and a bottom. -/
+variables [semilattice_sup P] [ideal_inter_nonempty P]
+variables (I J K : ideal P)
```

Comment:
> Maybe do the same thing here?
What do you mean here? It seems it is the case here.

Reply:
This is already fixed by your most recent commits. When it was `variables (I J K : ideal P)` instead of `variables {I J K : ideal P}`, the `lemma sup_le` had `I J K` as explicit parameters, and I was suggesting that they should be implicit parameters.